modules/apps/27-interchain-accounts/controller/types/keys.go

```diff
@@ -10,3 +10,6 @@ const (
 	// ParamsKey is the store key for the interchain accounts controller parameters
 	ParamsKey = "params"
 )
+
+// KeyControllerEnabled is the store key for ControllerEnabled Params for legacy purposes.
+var KeyControllerEnabled = []byte("ControllerEnabled")
```

Comment:
I can't see much reason for this to be left behind now?
Looking it up on this branch, it looks like it is only used in tests, which also tells me there might be more code to remove?